go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -4755,3 +4755,43 @@ var DoltTagTestScripts = []queries.ScriptTest{
 		},
 	},
 }
+
+var DoltRemoteTestScripts = []queries.ScriptTest{
+	{
+		Name:        "dolt-remote: SQL add remotes",
+		SetUpScript: []string{},
+		Assertions: []queries.ScriptTestAssertion{
+			{
+				Query:    "CALL DOLT_REMOTE('add','origin','file://../test')",
+				Expected: []sql.Row{{0}},
+			},
+			{
+				Query:    "SELECT name, IF(CHAR_LENGTH(url) < 0, NULL, 'not null'), fetch_specs, params FROM DOLT_REMOTES",
+				Expected: []sql.Row{{"origin", "not null", sql.MustJSON(`["refs/heads/*:refs/remotes/origin/*"]`), sql.MustJSON(`{}`)}},
+			},
+		},
+	},
+	{
+		Name: "dolt-remote: SQL remove remotes",
+		SetUpScript: []string{
+			"CALL DOLT_REMOTE('add','origin1','file://.')",
```

Comment:
Add an error case like `call dolt_remote()` or `call dolt_remote('origin')` or something, somewhere in the tests, just to get some coverage over the error handling code.